src/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+  <BrowserRouter>
```

Comment:
- [ ] [Optiona]try cleaning up your index.js file by removing the comments on this file :+1:

Reply:
Hi @VICTOR-CHUKWU , thanks for the review, I will follow your sugestions